app/views/stream/_main.erb

```diff
@@ -17,6 +17,7 @@
               <h4>
                 <%= exercise.problem.name %>
                 <span class="profile"> <%= exercise.username %> </span>
+                <span class="track-icon"><%= track_icon(exercise.problem.track_id, 20) %></span>
```

Comment:
@radar that will require changing the track_icon view helper and probably belongs in a different PR, will you create an issue for it please?